enumerables.rb

```diff
@@ -0,0 +1,37 @@
+module MyEnumerable
+  def all?
+    @list.each { |num| return false unless yield(num) }
+    true
+  end
+
+  def any?
+    @list.each { |num| return true if yield(num) }
+    false
+  end
+
+  def filter
+    result = []
+    @list.each { |num| result << num if yield(num) }
+    result
+  end
+end
+
+class MyList
+  include MyEnumerable
+
+  def initialize(the_list)
+    @list = the_list
+  end
+
+  def each
+    @list.each
+  end
+end
```

Comment:
- [ ] After separating the MyList class into its own variable, please remember to use the `require_relative` to import `MyEnumerable`  into the it since that is the best ruby convention. :+1: 
- [ ] I have checked and the code is working as expected. However, you will need to kindly check and make sure everything is working fine before submitting for review again since. :pray:

Reply:
Thank you,   
Well Noted!